lua/astrocommunity/pack/docker/docker.lua

```diff
@@ -0,0 +1,14 @@
+local utils = require "astronvim.utils"
+return {
+  {
+    "williamboman/mason-lspconfig.nvim",
+    opts = function(_, opts)
+      opts.ensure_installed =
+        utils.list_insert_unique(opts.ensure_installed, "docker-compose-language-service", "dockerfile-language-server")
```

Comment:
```suggestion
        utils.list_insert_unique(opts.ensure_installed, { "docker-compose-language-service", "dockerfile-language-server" })
```

this should be a list like table

Reply:
added as suggested